geomesa-gs-plugin/geomesa-hbase-gs-plugin/pom.xml

```diff
@@ -13,7 +13,7 @@
     <parent>
         <artifactId>geomesa-gs-plugin</artifactId>
         <groupId>org.locationtech.geomesa</groupId>
-        <version>1.2.0-SNAPSHOT</version>
+        <version>1.2.1-SNAPSHOT</version>
```

Comment:
is geomesa-hbase-gs-plugin not in the geomesa-gs-plugin pom? please add it if not.